website/docs/reference/resource-configs/meta.md

```diff
@@ -172,6 +173,31 @@ You can't add YAML `meta` configs for [generic tests](/docs/build/data-tests#gen
 
 </TabItem>
 
+<TabItem value="unit tests">
+
```

Comment:
@dbeatty10 - were unit tests introduced in 1.8? wondering if we should write up a 'Available from dbt 1.8 and higher' sentence here?